cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,21 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until database is available."""
+
+    def handle(self, *args, **options):
+        """Handle the command."""
+        self.stdout.write("Waiting for database...")
+        db_conn = False
+        while not db_conn:
+            try:
+                connections["default"].cursor()
+                db_conn = True
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
```

For even more robust error handling, you could consider adding a timeout to this loop. If there's a persistent configuration issue (like a wrong password or hostname), this script will currently wait forever. Limiting the retries (e.g., to 30 seconds) and then raising a `CommandError` would cause the container to fail fast, which can make debugging such problems easier.